lib/src/zap/rdma/zap_rdma.c

```diff
@@ -1099,6 +1099,7 @@ static int cq_event_handler(struct ibv_cq *cq, int count)
 {
 	struct ibv_wc wc;
 	int rc, poll_count = 0;
+	memset(&wc, 0, sizeof(wc));
```

Comment:
@baallan this doesn't do anything. A work completion does not go on the wire, it is a local data structure.

Reply:
ok mislabeling. apologies for the confusion. it does go into log formatting which causes use of uninit data to control output write logic.